JoonasHeinonen/escape-from-zordoom-prison
# Feature/assets lamp posts 0

### [Assets and audio enhancement #0](https://trello.com/c/mDh7NCBe/66-assets-and-audio-enhancement-0)

https://user-images.githubusercontent.com/33086486/221438434-c1c9ffe9-a86c-4bf9-b28d-12925b0a4ef0.mp4

**Changes:**
 - Created a lamp post to be a destructible entity.
 - Added a smoke effect when destroying a bolt crate.
 - Imported new block for the mesh library.
 - Imported an unarmed sprite sheet for Rivet.
 - Imported and inited the script for the ammo can.

**How to test:**
 - See that the lamp post can be destroyed and that it generates bolts.
 - See that a destroyed crate emits a smoke effect.
 - See that the new blocks look good.
 - See that the Rivet's sprite sheet is good
 - See that the ammo can emits the proper sound effect, its size is good and it emits the particle.
